pan/Makefile

```diff
@@ -6,7 +6,7 @@ IMAGE_GLOB=*.qcow2
 # match versions like:
 # PA-VNM-KVM-7.0.1.qcow2
 # PA-VM-KVM-10.0.6.qcow2
-VERSION=$(shell echo $(IMAGE) | sed -e 's/.*-\([0-9]\{1,2\}\.[0-9]\{1,2\}.[0-9]\{1,2\}\)\.qcow2$$/\1/')
+VERSION=$(shell echo $(IMAGE) | sed -e 's/.*-\([0-9]\{1,2\}\.[0-9]\{1,2\}.[0-9]\{1,2\}\(\|-h[0-9]\{1,2\}\)\)\.qcow2$$/\1/')
```

Comment:
for some platforms we started to move away from the arcane sed/grep expressions in favor of a more simple one where a user renames the image to have a simple versioning scheme. Like this:

https://github.com/hellt/vrnetlab/blob/master/huawei_vrp/Makefile#L9

it makes it easier for us (and users) to just rename the qcow to have a predefined prefix, and then format the version the way that want it to be